pinot-connectors/pinot-spark-3-connector/documentation/read_model.md

```diff
@@ -0,0 +1,140 @@
+<!--
```

Comment:
Full Disclosure: I copied this documentation from the Spark2 based implementation since at this point the functionality is pretty much same. So it is possible that I overlooked at some details, but always happy to fix.